code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -838,6 +886,7 @@
 /datum/tarot/reversed/the_tower
 	name = "XVI - The Tower?"
 	desc = "Creation brings destruction."
+	extended_desc = "makes stone containing minerals to shoot from the ground."
```

Comment:
I might be misunderstanding the code, but doesn't this just spawn some minerals?

```suggestion
	extended_desc = "conjures some random minerals on the floor."
```

Reply:
It's specifically the like, rocks on lavaland that you PKA open to get the minerals, no?

Reply:
it spawns the lavaland rock they can mine that has ores in it, yea